### PR TITLE
updating the DB schema

### PR DESCRIPTION
**Pull request by jorge-caballero**
hey @iloveicedgreentea  - did my best to update the files, but i couldn't find documentation to explain who to include defaults that are expression

### CODE REVIEW

**Comment by iloveicedgreentea** [comment]
Updated with default values if missing/null, updated column types, added GET `/api/v1/recentLocation`, added external_location_id

File: README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ GET `/api/v1/health`
 ### List locations
```
Comment (jorge-caballero) [inline README.md:8]: @iloveicedgreentea One of the FEMA folks called my attention to the fact that the document lacks any explicit mention of the base URL. I recognize that a savvy developer could figure it out, but it seems that not having it on the readme is a hurdle for some of the folks interested in our data

File: application/database/database_routes.py [hunk 2]
```diff
@@ -245,11 +275,16 @@ def get_location(location_id):
...
 def update_location(id):
+  """
+  Update a location
+  """
   response = make_response(jsonify("Not Implemented"), 501)
   response.headers.add('Access-Control-Allow-Origin', app.config['SITE_ENDPOINT'])
   response.headers.add('Access-Control-Allow-Headers', 'Authorization, Content-Type')
 
+  #todo: dont update created_on
+
```
Comment (jorge-caballero) [inline application/database/database_routes.py:289]: for implementation: the following fields are not allowed to be modified via the API: 
`created_on`
`record_id`
`location_id`

additionally, updates to the following fields should NOT be allowed to anyone outside our org, and (not via the FE)
`is_hidden`
`is_verified`
`deleted_on`
`external_location_id`

**Review by jorge-caballero — COMMENTED** [review]
@adimarco made some comments on changes made by @iloveicedgreentea. they amount to separate tasks that have not been spec'ed or prioritized.

File: README.md (hunk 1 above)
Comment (iloveicedgreentea) [reply README.md:8]: Updated readme with base url

**Pull request merged**